### ISSUE TITLE
Failure with @typescript-eslint/tslint plugin

### ISSUE THREAD

**Issue by imirkin** [webpack-contrib/eslint-webpack-plugin#149]
The relevant bits from package.json:

```json
    "@typescript-eslint/eslint-plugin": "5.20.0",
    "@typescript-eslint/eslint-plugin-tslint": "5.20.0",
    "@typescript-eslint/parser": "5.20.0",
    "codelyzer": "6.0.2",
    "eslint": "8.14.0",
    "eslint-webpack-plugin": "3.1.1",
    "tslint": "6.1.3",
```

These are all the latest at the time of writing. With a custom angular webpack.config.js, I have

```js
      new ESLintPlugin({
        files: ["src/**/*.ts", "src/**/*.html"]
      })
```

which seems to work, so long as I don't try to use the tslint adapter plugin. It seems that the `.eslintrc.json` file containing just

```json
      "plugins": [
        "@typescript-eslint/tslint"
      ],
      "rules": {
        "@typescript-eslint/tslint/config": ["error", {
          "lintFile": "./tslint.json"
        }]
      }
```

is enough to trigger this error:

> Error: Failed to load plugin '@typescript-eslint/tslint' declared in '.eslintrc.json#overrides[0]': Class extends value undefined is not a constructor or null

I know it's working at least a little bit, since it's loading the tslint.json file (as a strict JSON with comments disallowed for some reason, unlike what ends up happening if I go through Angular's "ng lint" -- very odd). But once I fixed those, which also came up as plugin load errors, I got the error above.

This seems to stem from `node_modules/@typescript-eslint/eslint-plugin-tslint/dist/custom-linter.js` which has:

```js
const tslint_1 = require("tslint");
// @ts-expect-error - We need to access the program, but Linter has private program already
class CustomLinter extends tslint_1.Linter {
```

Given that this all works via angular's invocation of eslint (`ng lint`), I'm assuming there's some sort of issue with how `eslint-webpack-plugin` loads things. Let me know if this issue needs to be redirected to the `@typescript-eslint/eslint-plugin-tslint` maintainers.

The full error/stacktrace is:

```
TypeError: Class extends value undefined is not a constructor or null
    at Object.<anonymous> (/path/to/node_modules/@typescript-eslint/eslint-plugin-tslint/src/custom-linter.ts:5:35)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/path/to/node_modules/@typescript-eslint/eslint-plugin-tslint/src/rules/config.ts:4:1)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
```

**Comment by ricardogobbosouza** [comment]
Hi @imirkin 
Could you provide a repository?
It looks like it's not a problem with this plugin but with the linter
